samples/04_gis_analysts_data_scientists/forecasting_monthly_rainfall_in_California_using_deeplearning_timeseries_model_from_arcgis_learn.ipynb

```diff
@@ -0,0 +1,2195 @@
+{
```

Comment:
<p>The dataset above has 480 data samples, each representing the monthly rainfall of California over 40 years (1980-2019). Out of these 40 years of data, 39 years (1980-2019) will used for training the model, and the remaining year of data will be held for validation. As such, the dataset is now split into training and test datasets, as seen in the following cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p><em>Grammar edit:</em></p><p> </p><p>Out of these 40 years of data, 39 years (1980-2019) <strong>will be used</strong> for training the model, and the remaining year of data will be held for validation.</p>